### DISCUSSION TITLE
Local function RegisterServices

### DISCUSSION THREAD

**Discussion by ShervanN** [ProfessionalCSharp/ProfessionalCSharp2021#183] [General]
Hi,

On page 400,  CHAPTER 15 Dependency Injection and Configuration, In Using Scoped Services you wrote:

> Let’s start registering services with the local function **_RegisterServices_**. ServiceA is registered as a scoped service, ServiceB as singleton, and ServiceC as transient (code file DI/ServicesLifetime/Program.cs):

I didn't see any function or local function with RegisterServices name in the code.

**Comment by christiannagel** [comment]
The method name was changed, and missed updating it in the text. Thanks! 

Updated 
https://github.com/ProfessionalCSharp/ProfessionalCSharp2021/blob/main/Updates.md
